src/summary/history/allergy-card-level-two.component.tsx

```diff
@@ -26,7 +26,7 @@ export function AllergyCardLevelTwo(props: AllergyCardLevelTwoProps) {
 
       return () => abortController.abort();
     }
-  }, [patient.identifier[0].value]);
+  }, [isLoadingPatient, patient]);
 
```

Comment:
why did you add `isLoadingPatient` here?

Reply:
It is a dependency on this React useEffect.